standard/links/commons/links.lua

```diff
@@ -211,6 +215,7 @@ function Links.transform(links)
 		toornament = links.toornament,
 		toornament2 = links.toornament2,
 		toornament3 = links.toornament3,
+		['trackmania-io'] = links['trackmania-io'] or links.trackmania_id,
```

Comment:
Spontanously my thought is that this should be in the Custom instead (eg. see Dota2 Team or Player)

Reply:
1) Vogan wanted them all in the commons version unless, the /Custom was only intended for wiki specific overwrites
2) the `Links.transform` function is currently not overwritable (the custom one only sets data for the prefixes/suffixes tables)

Reply:
I'm assuming that `trackmania_id` would be a parameter that have multiple uses, including of them being an ID for trackmania-io? If so, it would be similiar to this
https://liquipedia.net/dota2/index.php?title=Module:Infobox/Person/Player/Custom&action=edit#mw-ce-l72

Reply:
ah you were only reffering to `links.trackmania_id`
yeah i can kick that^^